engine/src/main/resources/org/hibernate/validator/ValidationMessages.properties

```diff
@@ -28,7 +27,8 @@ org.hibernate.validator.constraints.Range.message                   = must be be
 org.hibernate.validator.constraints.SafeHtml.message                = may have unsafe html content
 org.hibernate.validator.constraints.ScriptAssert.message            = script expression "{script}" didn't evaluate to true
 org.hibernate.validator.constraints.URL.message                     = must be a valid URL
+org.hibernate.validator.constraints.NumberPattern.message           = must match "{regexp}"
```

Comment:
it'll be a longer message than, as that format parameter is optional and we would need to somehow mention that right ? So for example it might be something like:
`String representation of a number, formatted with {numberFormat} (if one was provided)  must match "{regexp}"`

But I don't know ...